python/GafferUI/MultiLineTextWidget.py

```diff
@@ -46,8 +46,9 @@
 class MultiLineTextWidget( GafferUI.Widget ) :
 
 	WrapMode = IECore.Enum.create( "None", "Word", "Character", "WordOrCharacter" )
+	Role = IECore.Enum.create( "Text", "Code" )
```

Comment:
I could see that happening, but I think even then a Code role might be a useful catch-all fallback. I also suspect a full on editor with syntax highlighting might mean a new class (or derived class perhaps). This was just a quick way of making a minor improvement to the look of things, and the Role concept meant the style itself was free to change independently of the code/metadata. I'm thinking the Role concept could apply pretty well in many similar scenarios where we've either added specific API (e.g. `TextWidget.displayMode()`) or hardcoded a bunch of stylesheet hacks (e.g. SceneInspector frame styles). It seems like a reasonable way for the code to say "the purpose of this doohickey is X", and give the style the freedom to represent that how it wants.